tests/test_graindata.py

```diff
@@ -7,41 +7,86 @@
 from hexrd.cli.fit_grains import GrainData
 
 
-save_file = "save.npz"
+@pytest.fixture
+def exp90():
+    return (np.pi/2) * np.identity(3)
+
 
-exp90 = (np.pi/2) * np.identity(3)
-rmats90 = np.array([
-    [[1, 0, 0], [0, 0,- 1], [0, 1, 0]],
-    [[0, 0, 1], [0, 1, 0], [-1, 0, 0]],
-    [[0, -1, 0], [1, 0, 0], [0, 0, 1]]
-])
+@pytest.fixture
+def quats90():
+    c45, s45 = np.cos(np.pi / 4), np.sin(np.pi /4)
+    return [[c45, s45, 0, 0], [c45, 0, s45, 0], [c45, 0, 0, s45]]
```

Comment:
Constants could also just be global variables like `QUATS_90`. pytest fixtures often use other things like environment variables or data on disk to make them. But it doesn't hurt to do it this way, and you can keep it.